lti_consumer/tests/unit/test_api.py

```diff
@@ -184,49 +231,17 @@ def test_retrieve_with_block(self):
                     lti_config.lti_1p3_client_id
                 ),
                 'deep_linking_launch_url': 'https://example.com',
-                'deep_linking_content_items': []
```

Comment:
This assert uses assertCountEqual, but that won't check everything we need it to. When comparing dicts, it only looks at the keys. This assert would pass: self.assertCountEqual({"a": 1}, {"a": 2}).

Don't we want assertEqual for these tests?

Reply:
@nedbat You're right. This test also had a few incorrect values and I fixed those as well.

Reply:
Thanks!